modules/imgproc/src/thresh.cpp

```diff
@@ -764,6 +764,12 @@ thresh_32f( const Mat& _src, Mat& _dst, float thresh, float maxval, int type )
     CV_IPP_CHECK()
     {
         IppiSize sz = { roi.width, roi.height };
+
+        int *thresh_hex = reinterpret_cast<int*>(&thresh);
+        int onebit = 0x00000001;
+        (*thresh_hex) += onebit;
```

Comment:
Please check implementations of [`std::nextafter`](https://en.cppreference.com/w/cpp/numeric/math/nextafter)

Add test with mantissa value equal to 111..11b (all 23 bits are '1')

Reply:
Fixed